plugins/supply-depot

```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/Naughty-Klaus/supply-depot.git
+commit=ee716a1f69e1c19df30ea633cefcbf3177890ae8
+warning=This plugin submits your username and IP address and in game stats to a 3rd party server not controlled or verified by the RuneLite Developers.
```

Comment:
```suggestion
warning=This plugin submits your username, IP address, and in game stats to a 3rd party server not controlled or verified by the RuneLite Developers.
```
nitpick from NFC's suggestion

Reply:
Actually ingame stats aren't transmitted--I apparently copied the wrong warning from another plugin to use as an example here. Instead, I'll commit this:
```suggestion
warning=This plugin submits your username and IP address to a 3rd party server not controlled or verified by the RuneLite Developers.
```

Reply:
Alright thank you, I also agree with Felanbird. Grammar really is important.